frontend/src/components/AGComponente/AGComponent.css

```diff
@@ -0,0 +1,67 @@
+.AGModul {
+  position: relative;
```

Comment:
ist das 'position: relative' hier wirklich nötig, du verschiebst hier dann ja nicht von der Ausgangsposition aus.

Reply:
Das position relative ist wichtig. Begründung: Css Doku: "An element with position: absolute; is positioned relative to the nearest positioned ancestor (instead of positioned relative to the viewport, like fixed)."

Der Button is absolut positioned. Setzt man beim Modul kein position attribut, ist es kein positioned element und der button orientiert sich am document body (der button ist dann unten links in der ecke der seite)

Reply:
Muss ich leider sagen ist keine schöne Lösung @Max.

Wäre cool, wenn du die AGModule mit flexboxen bauen könntest. Das ist moderner. aber so funktioniert es im moment ja auch, kannst du also meinetwegen auch lassen. darfst du entscheiden